Gruntfile.js

```diff
@@ -0,0 +1,128 @@
+module.exports = function (grunt) {
+    // Project configuration.
+    grunt.initConfig({
+        modx: grunt.file.readJSON('_build/config.json'),
+        sshconfig: grunt.file.readJSON('/Users/jako/Documents/MODx/partout.json'),
```

Comment:
Yes. I have no better idea a the moment.

I use grunt-ssh to upload the minified css/js files and exclude them from automatic deployment in PhpStorm because of the duration of the build process to avoid changed files warnings. This part could be removed there for sure.